src/accountsdb/db.zig

```diff
@@ -122,7 +123,7 @@ pub const AccountsDB = struct {
     bank_hash_stats: RwMux(BankHashStatsMap) = RwMux(BankHashStatsMap).init(.{}),
 
     metrics: AccountsDBMetrics,
-    logger: Logger,
+    logger: ScopedLogger(@typeName(@This())),
```

Comment:
can you change this to self

Reply:
4d4ed66ea7e9138e47053e83a1ab6cbd2f515fcc